packages/office-ui-fabric-react/src/components/Breadcrumb/Breadcrumb.visualtest.ts

```diff
@@ -0,0 +1,17 @@
+import { Casper, IPhantomCSS } from '../../visualtest/PhantomCssInterface';
+import { baseUrl } from '../../common/VisualTest';
+declare var phantomcss: IPhantomCSS;
+declare var casper: Casper;
+/* tslint:disable:no-function-expression */
+casper.start(baseUrl + 'breadcrumb').
+  then(function () {
+    phantomcss.screenshot('.Breadcrumb', 'Breadcrumb_not_pressed');
+  }).then(function () {
+    this.mouse.move('.Breadcrumb');
+    phantomcss.screenshot('.Breadcrumb', 'Breadcrumb_hovered');
```

Comment:
@gitjain you can pass any selector you want into this.move.mouse. Even if you need to use nth-child or something, you should be able to target something to hover over and test.